index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Adaobi Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+    <nav class="container">
+      <a href="#" class="navbar-logo">Adaobi Okoye</a>
+      <a href="#" class="navbar-menu">
+        <img src="/Icons/Union.png" alt="Menu"/>

```

Comment:
kindly change the location of your files. instead of `/Icons/Union.png"` use `icons/Union.png"` and it will work